### DISCUSSION TITLE
Bring TikTok Shop Orders into Sellbrite

### DISCUSSION THREAD

**Discussion by eversshop** [Sellbrite/Sellbrite-API#29] [Q&A]
Hi, has anyone been successful bringing TikTok Shop orders into Sellbrite?

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

Using the API we currently don’t have the option to import orders into Sellbrite, you can only export orders out. You can review our endpoints from here: https://developer.sellbrite.com/endpoints/orders
